changelog.txt

```diff
@@ -13,6 +13,7 @@ V1.XX.X
 [QMS-31] Fix all links to Bitbucket in the code
 [QMS-53] Unload Garmin `Archive` when folder is deflated
 [QMS-54] Invalid point cannot be deactivated when tracks are read from the navigation device
+[QMS-58] Windows Build and Installer: use PROJ 6.2
```

Comment:
Can you do me a favour and change V1.XX.X into V1.14.0 and open a new V1.XX.X section with your ticket. I forgot to do that on the release.

Reply:
Oh sure, I should have seen this by myself.
I made the change, amended to the commit and "push --force"'d.
Please check again.